tests/AbstractFixerTest.php

```diff
@@ -89,7 +88,7 @@ public function testGetWhitespacesFixerConfig(): void
 
         $fixer->setWhitespacesConfig($newConfig);
 
-        $config = $fixer->getWhitespacesConfig();
+        $config = $fixer->extraBehavior();
```

Comment:
can we keep here `->getWhitespaceConfig()`?

Reply:
Any suggestion on how to do it, better than having 3 interfaces instead of `FixerDoubleInterface`?

Reply:
We have well defined interfaces already, my solution would be to drop `FixerDoubleInterface`.
I believe having 3 interfaces is better than having one interface with a method of totally different behaviours, depending on context.

Imagine you would like a test with a double that is having both, `whitespaceConfig` and `configurationResolver`. You would add §extraBehaviour2` ?